cylc/uiserver/handlers.py

```diff
@@ -246,9 +246,9 @@ def set_default_headers(self) -> None:
     @web.authenticated
     # @authorised  TODO: I can't think why we would want to authorise this
     def get(self):
-        user_info = {
-            'name': get_username(self)
-        }
+        user_info = dict(self.current_user.__dict__)
+
+        user_info['name'] = get_username(self)
```

Comment:
It looks like Jupyter Server already provides a "name" field, so this is overwriting that field.

We could consider removing this line:

```suggestion
```

However, we may want to update the UI to use `usename` rather than `name`.

Note, that the two fields appear to be the same by default:

```python
>>> from jupyter_server.auth import User
>>> User('foobar')
User(username='foobar', name='foobar', display_name='foobar', initials=None, avatar_url=None, color=None)
```